webclient/api/src/main/java/io/helidon/webclient/api/Proxy.java

```diff
@@ -667,6 +677,19 @@ public Builder type(ProxyType type) {
             return this;
         }
 
+        /**
+         * Forces HTTP CONNECT with the proxy server.
+         * Otherwise it will not execute HTTP CONNECT for HTTP protocol.
```

Comment:
This is not exact. Please reword it to make it clear that we talk about plain vs. TLS.
Both are HTTP protocol.

Reply:
Done